cloud/amazon/s3.py

```diff
@@ -131,9 +131,13 @@
     version_added: "2.0"
   s3_url:
     description:
-      - S3 URL endpoint for usage with Eucalypus, fakes3, etc.  Otherwise assumes AWS
+      - S3 URL endpoint for usage with Ceph, Eucalypus, fakes3, etc.  Otherwise assumes AWS
     default: null
     aliases: [ S3_URL ]
+  rgw:
+    description:
+      - Enable Ceph RGW S3 support
+    default: false
```

Comment:
From bcoca: If this option is specific to ceph then there needs to be an indication in the docs that it is ignored when used against other S3 compatible endpoints.